crates/ruff_linter/src/checkers/ast/mod.rs

```diff
@@ -889,35 +914,6 @@ impl<'a> Visitor<'a> for Checker<'a> {
         };
 
         // Step 3: Clean-up
```

Comment:
Nit: should we remove this, and change the next comment ("Step 4" to "Step 3")?

Reply:
I actually did that initially, and then I reverted it because the "four steps" are a consistent and common pattern throughout this file, and they are always the same, so I thought it better to leave the empty marker and stay consistent.

Reply:
But I can remove the empty step and re-number if you prefer that

Reply:
No, on second thought, you're right!